src/patterns/PaymentForm.vue

```diff
@@ -0,0 +1,523 @@
+<template>
```

Comment:
It does not look like this file has had tokens added or style used to replace the props.